selfdrive/car/volkswagen/mqbcan.py

```diff
@@ -125,11 +125,11 @@ def create_acc_accel_control(packer, bus, acc_type, acc_enabled, accel, acc_cont
   return commands
 
 
-def create_acc_hud_control(packer, bus, acc_hud_status, set_speed, lead_distance):
+def create_acc_hud_control(packer, bus, acc_hud_status, set_speed, lead_distance, distance):
   values = {
     "ACC_Status_Anzeige": acc_hud_status,
     "ACC_Wunschgeschw_02": set_speed if set_speed < 250 else 327.36,
-    "ACC_Gesetzte_Zeitluecke": 3,
+    "ACC_Gesetzte_Zeitluecke": distance + 2,
```

Comment:
@jyoung8607 can you test this? Does cycling between 3, 4, 5 bars seem like a good experience to you?

Reply:
I'm away from my cars and can't test for a couple days. From memory, the cluster continues to display the bars in sort of a dimmed/grayed-out state when disengaged. The only time the bars go away is main switch off. I think it's likely to be a decent experience as-written, but I will test and advise.